OpenRA.Mods.Common/Widgets/Logic/Ingame/ObserverStatsLogic.cs

```diff
@@ -533,6 +536,11 @@ string AverageOrdersPerMinute(double orders)
 			return (world.WorldTick == 0 ? 0 : orders / (world.WorldTick / 1500.0)).ToString("F1");
 		}
 
+		string Vision(int revealedCells)
+		{
+			return (revealedCells / (float)world.Map.ProjectedCells.Length * 100).ToString("F0") + "%";
```

Comment:
IMO if you have at least 1 cell of vision it should show 1%, It feels rather weird when this metric is 0%

Reply:
I think this applies to 100% too. Perhaps we could add a special case `<1%`/`>99%` values and don't show 0% and 100% unless it is exactly that.

Reply:
Rounding always up now.